src/test/java/ch/liquidmind/inflection/grammar/CompilationUnitTest.java

```diff
@@ -0,0 +1,107 @@
+package ch.liquidmind.inflection.grammar;
+
+import org.junit.Test;
+
+import ch.liquidmind.inflection.compiler.util.InflectionCompilerTestUtility;
+import ch.liquidmind.inflection.test.AbstractInflectionTest;
+
+public class CompilationUnitTest extends AbstractInflectionTest
+{
+
+	@Test
+	public void testCompilationUnit_Empty_SuccessfulCompilation() throws Exception
+	{
+		doTest( job -> {
+			InflectionCompilerTestUtility.assertSuccessfulCompilation( job );
+		} , createInflectionFileMock( "" ) );
+	}
+
+	@Test
+	public void testCompilationUnit_WhiteSpace_SuccessfulCompilation() throws Exception
+	{
+		doTest( job -> {
+			InflectionCompilerTestUtility.assertSuccessfulCompilation( job );
+		} , createInflectionFileMock( " " ) );
+	}
+
+	@Test
+	public void testCompilationUnit_MultipleTaxonomies_SuccessfulCompilation() throws Exception
```

Comment:
Does this belong in TaxonomyTest? Or is it redundant? Check rest of methods in this class as well.